cmd/neofs-node/main.go

```diff
@@ -89,6 +91,10 @@ func initApp(c *cfg) {
 	initAndLog(c, "control", initControlService)
 
 	initAndLog(c, "morph notifications", listenMorphNotifications)
+
+	c.workers = append(c.workers, newWorkerFromFunc(func(ctx context.Context) {
+		go c.configWatcher()
```

Comment:
Why `go`? Workers are separate go-routines by design.

Also if `configWatcher` is stopped by context it's worth to make the function to accept the context as a parameter.

Reply:
forgot what workers are

fixed